lib/webapp.js

```diff
@@ -233,21 +233,40 @@ const staticHandlers = {
         but this should be enforced somehow in the future.
       */
       if (!type) {
-        do404(req.url, res, "A plugin type must be specified");
-        return;
+        type = "all"
       }
       const acceptLanguage = 
         translationUtils.getAcceptLanguageFromCookies(req.cookies) || req.headers['accept-language'] || '';
       const pluginDefs = plugins.map(p => p.exportTranslatedDef(acceptLanguage));
+            
+      let filteredPluginDefs = []
+      let allowedPlugins;
+
+      if (options.serverConfig.dataserviceAuthentication.rbac) {
+        if (req.username) {
+          allowedPlugins = configService.getAllowedPlugins(configService.CONFIG_SCOPE_USER, options, req.username);
+        } else {
+          allowedPlugins = configService.getAllowedPlugins(configService.CONFIG_SCOPE_PRODUCT, options);
```

Comment:
Not product, try CONFIG_SCOPE_INSTANCE, because thats the lowest level that doesnt require a username.